OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
```

Comment:
This will have an effect on all the nested OWNERS files.

If this is not what you want, then adding no_parent_owners: true to nested OWNERS will fix it.

Reply:
Also, in case some areas/components absolutely (and always) require a person to review it (but not approve) you can define "required reviewers".

[This struct](https://github.com/kubernetes/test-infra/blob/ae26658abae0b1213549d1326b3b252422fe0215/prow/repoowners/repoowners.go#L240) helped me in the past :)

While [this one](https://github.com/kubernetes/test-infra/blob/ae26658abae0b1213549d1326b3b252422fe0215/prow/repoowners/repoowners.go#L47) is about the `no_parent_owners` thing.

Reply:
Also, in case some areas/components absolutely (and always) require a person to review it (but not approve) you can define `required_reviewers` key.

[This struct](https://github.com/kubernetes/test-infra/blob/ae26658abae0b1213549d1326b3b252422fe0215/prow/repoowners/repoowners.go#L240) helped me in the past :)